src/matchmaker/drawingAlgorithms.js

```diff
@@ -36,16 +36,25 @@ function separateTopFromBottomPlayers(players) {
 //Algorithms for drawing later round
 // pairPlayersLaterRound : [players] -> [pairings]
 function pairPlayersLaterRound(players) {
+  //this is the "safest" algorithm
   const basicDrawing = basicDrawingAlgorithm(players);
-  const drawing = advancedDrawing(players);
-  return basicDrawing;
+  if (basicDrawing !== false) {
```

Comment:
kann man das schöner lösen?
Dass zieht sich durch den ganzen PR dass ich einfach ein **false** returne wenn eine function fehlschlägt.
Tut sie dass soll halt ein anderer algorithmus probiert werden

Reply:
du könntest einfach ein Ergebnis zurückgeben und falls sie fehlschlägt gibst du eben `null` oder ein leeres Array zurück (je nachdem wie dein Ergebnis sonst aussieht). mit `if (basicDrawing) { ... }` testest du dann, ob es ein Ergebnis gab. `undefined`, `null`, `false` etc zählen alle als *falsy*, werden konkret in einer boolean expression als `false` angesehen.

Reply:
hmm hast auch keine coole Lösung :P